Replicator/Puller.cc

```diff
@@ -350,7 +350,7 @@ namespace litecore { namespace repl {
     Worker::ActivityLevel Puller::computeActivityLevel() const {
         ActivityLevel level;
         if (_fatalError || !connection()) {
-            level = kC4Stopped;
+            level = _unfinishedIncomingRevs > 0 ? kC4Busy : kC4Stopped;
```

Comment:
I like the logic, but I think it'd be clearer if done by moving the existing `_unfinishedIncomingRevs` test to the start of the `if` chain:

```c++
if (_unfinishedIncomingRevs > 0) {
    level = kBusy;
} else if (_fatalError || !connection()) {
    level = kStopped;
} else ...